research/conditional/utils/argparse.py

```diff
@@ -71,7 +71,7 @@ def introduce_parser_arguments(
     )
     parser.add_argument("--learning_rate", type=math_eval, required=True)
     parser.add_argument("--final_lr_step", type=int, required=False)
-    parser.add_argument("--lr_warmup_percent", type=float, required=False)
+    parser.add_argument("--lr_warmup_fraction", type=float, required=False)
```

Comment:
pls change the name to `lr_warmup_fraction_in_steps` to make it easy to see that the fraction is not counted in tokens